content/workshops/jboss-migration-toolkit/lab2-using-jmt-cli.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Lab 2 - Using RHMAT CLI
+workshops: jboss-migration-toolkit
+workshop_weight: 20
+layout: lab
+---
+
+# About Red Hat Application Migration Toolkit CLI
+The CLI is a command-line tool in the Red Hat Application Migration Toolkit that allows users to assess and prioritize migration and modernization efforts for applications. It provides numerous reports that highlight the analysis results. The CLI is for advanced users with highly customized needs looking for fine-grained control of RHAMT analysis options or to integrate with external automation tools. 
+
+{{% alert warning %}}
+These instructions are only when you are doing this lab outside of the controlled workshop.  The workshop environment has downloaded, installed, and configured your student environments prior to commencement of this workshop session.
+{{% /alert %}}
+
+# Set Up
+1. Install the Java SE Development Kit (JDK) version 8. We recommend using the OpenJDK or the Oracle JDK.
+
+2. Install the RHAMT CLI.
+
+  a. Download the [ZIP archive][1].
+  b. Extract the ZIP archive.
+
+3. Download the sample application, [jee-example-app-1.0.0.ear][2], to your file system.
+
+~~~~
+$ unzip migrationtoolkit-rhamt-cli-4.0.0.Beta2.1.zip
+~~~~
+
+# Step 1. Analyze your first application
+Now that the RHAMT CLI is installed, you’re ready to try it out. We’ll show you how to run the CLI to analyze a sample WebLogic application to identify the changes necessary to migrate it to JBoss EAP 7.
+
+> <i class="fa fa-terminal"></i> Navigate to the URI provided by your instructor and login with the user/password provided
+
+1. Escalate your user privileges.
+
+~~~
```

Comment:
This needs to be indented under your numbering. It's currently breaking your numbering and being as `1.` for each step below.

In other words, it's being rendered as:
```
1. your text
your code
1. other text that should have a 2 in front
your other code
```